.remarkrc.js

```diff
@@ -0,0 +1,18 @@
+exports.settings = { bullet: '*', paddedTable: false };
+
+exports.plugins = [
+    'remark-frontmatter',
+    'remark-gfm',
+    'remark-preset-lint-recommended',
+    'remark-preset-lint-consistent',
+    ['remark-lint-list-item-indent', 'space'],
+    ['remark-lint-no-dead-urls', false], // Does not work because of github protection system
+    ['remark-lint-maximum-line-length', 120],
```

Comment:
```suggestion
    ['remark-lint-maximum-line-length', 80],
```

Does it ignore long URLs?

Reply:
Applied locally.